README.md

```diff
@@ -1,3 +1,12 @@
+# prefect-databricks
+
+## Welcome!
+
+Prefect integrations interacting with Databricks
+
+The tasks within this collection were created by a code generator using the service's OpenAPI spec.
+
+The service's REST API documentation can be found [here](https://docs.databricks.com/dev-tools/api/latest/index.html).
```

Comment:
We can remove this section. The information regarding generated tasks is in the __Resources__ section.
```suggestion
```